flutter_auth_ui/android/src/main/java/com/dr1009/app/flutter_auth_ui/FlutterAuthUiPlugin.java

```diff
@@ -87,57 +84,54 @@ public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
     }
 
     @NonNull
-    private final PluginRegistry.ActivityResultListener listener =
-            new PluginRegistry.ActivityResultListener() {
-                @Override
-                public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
-                    if (requestCode != RC_SIGN_IN && requestCode != RC_EMAIL_LINK) {
-                        return false;
-                    }
-
-                    if (result == null) {
-                        // error
-                        return true;
-                    }
-
-                    if (resultCode == Activity.RESULT_OK) {
-                        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
-                        result.success(user != null);
-                        result = null;
-
-                        return true;
-                    }
-
-                    if (possibilityEmailLink) {
-                        // It may be that EmailLink is being handled, so it is not an error.
-                        return true;
-                    }
-
-                    result.error(String.valueOf(resultCode), "error result", "canceled action");
-                    return true;
-                }
-            };
+    private final PluginRegistry.ActivityResultListener listener = new PluginRegistry.ActivityResultListener() {
+        @Override
+        public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
+            if (requestCode != RC_SIGN_IN && requestCode != RC_EMAIL_LINK) {
+                return false;
+            }
+
+            if (result == null) {
+                // error
+                return true;
+            }
+
+            if (resultCode == Activity.RESULT_OK) {
+                FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
+                result.success(user != null);
+                result = null;
+
+                return true;
+            }
+
+            if (possibilityEmailLink) {
+                // It may be that EmailLink is being handled, so it is not an error.
+                return true;
+            }
+
+            result.error(String.valueOf(resultCode), "error result", "canceled action");
+            return true;
+        }
+    };
```

Comment:
```suggestion
    private final PluginRegistry.ActivityResultListener listener =
            new PluginRegistry.ActivityResultListener() {
                @Override
                public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
                    if (requestCode != RC_SIGN_IN && requestCode != RC_EMAIL_LINK) {
                        return false;
                    }

                    if (result == null) {
                        // error
                        return true;
                    }

                    if (resultCode == Activity.RESULT_OK) {
                        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
                        result.success(user != null);
                        result = null;

                        return true;
                    }

                    if (possibilityEmailLink) {
                        // It may be that EmailLink is being handled, so it is not an error.
                        return true;
                    }

                    result.error(String.valueOf(resultCode), "error result", "canceled action");
                    return true;
                }
            };
```